contracts/payment/PullPayment.sol

```diff
@@ -34,6 +48,9 @@ contract PullPayment {
      * @dev Called by the payer to store the sent amount as credit to be pulled.
      * @param dest The destination address of the funds.
      * @param amount The amount to transfer.
+     *
+     * Funds stored this way are sent to an intermediate {Escrow} contract, and
+     * can only be withdrawn from there to be sent to the payee.
```

Comment:
I think this is confusing. It can be understood as "people must withdraw their funds from the escrow", but there is a `withdrawPayments` function in this contract that can be used as well.